templates/corporate/billing.html

```diff
@@ -20,165 +20,183 @@
+                                        <strong>${{ renewal_amount }}</strong>.
+                                        {% else %}
+                                        Your plan would be downgraded to Zulip Free on <strong>{{ renewal_date }}</strong>. Please add a payment method
+                                        to continue on {{ plan_name }} after Free Trial.
+                                        {% endif %}
+                                    {% elif downgrade_at_end_of_cycle %}
+                                        Your plan will be downgraded to <strong>Zulip Limited</strong> on <strong>{{ renewal_date }}</strong>.
+                                    {% elif switch_to_annual_at_end_of_cycle %}
+                                        Your plan will be switched from monthly to annual billing on <strong>{{ renewal_date }}</strong>.
+                                    {% else %}
+                                        Your plan will renew on <strong>{{ renewal_date }}</strong> for
+                                        <strong>${{ renewal_amount }}</strong>.
+                                    {% endif %}
+                                {% else %}
+                                    Your plan ends on <strong>{{ renewal_date }}</strong>, and does not renew.
                                 {% endif %}
-                            </form>
+                            </p>
                         </div>
-                        <div id="cardchange-loading">
-                            <div class="zulip-loading-logo">
-                                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 773.12 773.12">
-                                    <circle cx="386.56" cy="386.56" r="386.56"/>
-                                    <path d="M566.66 527.25c0 33.03-24.23 60.05-53.84 60.05H260.29c-29.61 0-53.84-27.02-53.84-60.05 0-20.22 9.09-38.2 22.93-49.09l134.37-120c2.5-2.14 5.74 1.31 3.94 4.19l-49.29 98.69c-1.38 2.76.41 6.16 3.25 6.16h191.18c29.61 0 53.83 27.03 53.83 60.05zm0-281.39c0 20.22-9.09 38.2-22.93 49.09l-134.37 120c-2.5 2.14-5.74-1.31-3.94-4.19l49.29-98.69c1.38-2.76-.41-6.16-3.25-6.16H260.29c-29.61 0-53.84-27.02-53.84-60.05s24.23-60.05 53.84-60.05h252.54c29.61 0 53.83 27.02 53.83 60.05z"/>
-                                </svg>
+                        <div class="tab-pane" id="payment-method" data-email="{{stripe_email}}" data-key="{{publishable_key}}">
+                            <div id="cardchange-error" class="alert alert-danger"></div>
+                            <div id="cardchange-input-section">
+                                <form id="cardchange-form">
+                                    <p>Current payment method: <strong>{{ payment_method }}</strong></p>
+                                    {% if charge_automatically %}
+                                        {% if realm_on_free_trial and not payment_method_attached %}
```

Comment:
Should we write this `if` (and perhaps in some other relevant places) to account for the case where by customer request we remove their credit card in Stripe and they have to add a new payment method? Or is that case already reasonably handled in a different way?

Reply:
This was the UI for realms that had their card removed.

![Screenshot from 2020-11-17 14-07-43](https://user-images.githubusercontent.com/7190633/99371826-6ca05200-28e5-11eb-8971-5bbfd8dee998.png)

I have added a commit to replace "Update card" labels with "Add card" at the end. The functionality remains the same though. (ie, Clicking on the button would allow them to add a new card.)


![Screenshot from 2020-11-17 14-14-34](https://user-images.githubusercontent.com/7190633/99371948-90639800-28e5-11eb-87dc-bf2331d43329.png)